frontend/src/components/mspcontracts/BeneficiaryMemberDetails.vue

```diff
@@ -0,0 +1,17 @@
+<template>
```

Comment:
I like that this is it's own component but I feel that it's somewhat arbitrary to have the rows in the component but not the whole table.